lib/internal/modules/utils.js

```diff
@@ -0,0 +1,55 @@
+'use strict';
```

Comment:
When moving code from one file to another, please consider PR comments identifying what changed as doing so contravenes the diff.

I'm a bit late to the party (my invite seems to have got lost in the mail), but I would generally avoid a junk-drawer file, especially when creating it breaks the blame. Very few people know how and why this moved code exists the way that it does, and breaking the diff further silos that knowledge (arguably an XY problem, but here we are): now it's effectively impossible to find the right tribes(wo)man to ask.

Reply:
I thought about trying to preserve the git history/blame; that would have been a reason to push harder for a revert rather than a follow-up fix (my https://github.com/nodejs/node/pull/45923 will put most of `run_main.js` under my name, which isn’t really a reflection of reality) but I think we’re likely to lose most history soon anyway with the snapshot refactor.